content/resources/gen3-features.md

```diff
@@ -0,0 +1,83 @@
+---
+identifier: gen3-features
+title: Gen3 Features
+linktitle: /resources/gen3-features
+layout: withtoc
+menuname: featuresMenu
+date: 2019-12-04T10:57:21-05:00
+---
+
+
+# A list of Gen3 features
+
+Gen3 is an open-source platform that enables the standing-up of data commons for managing, analyzing, and sharing research data. Below you can find an exhaustive list of Gen3 features.
+
+## Dictionary and structured data
+
+* Explore the graph in the dictionary page of Windmill:
+    * Search keywords
+    * Visualize possible paths between nodes
+    * Download JSON or TSV format for node templates 
+* Explore project graphs and nodes in Windmill:
+    * Visualize graph model
+    * Browse/download data for each node
+* Structured data submission (create, update, delete records) in JSON, TSV or CSV:
+    * Submission page of Windmill
+    * Gen3 SDK (Python, R)
+* Export structured data as a JSON, TSV or CSV file
+* Structured data is stored in Postgres databases
+* ElasticSearch:
+    * Structured data can be stored in ElasticSearch by using our ETL tool
```

Comment:
Just my thoughts:  this could change to `Structured data can be flattened and stored in ElasticSearch by using our ETL tool`, and then its parent bullet `ElasticSearch` could change to `flattened data`.  Sorry I should probably also change this in our wiki repo, I will do that today :)

Reply:
Thank you for the feedback, Qingya!